test/library/standard/List/listRefBugMultiLoc.skipif

```diff
@@ -0,0 +1 @@
+CHPL_COMM==none
```

Comment:
You could drop this skipif since the test should still be valid and work on a single locale.

Reply:
Sounds good.